console/console-init/ui/src/modules/iot-project-detail/components/IoTProjectDetailHeader/IoTProjectDetailHeader.tsx

```diff
@@ -25,7 +25,7 @@ interface IIoTProjectDetailHeaderProps {
   type?: string;
   status?: string;
   isEnabled?: boolean;
-  changeEnable: () => void;
+  changeEnable: (checked: boolean) => void;
```

Comment:
Can we try event name changeStatus instead of changeEnable.

Reply:
Done!